apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js

```diff
@@ -0,0 +1,122 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+import './public-path';
+
+/**
+ * WordPress dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { Fill, Guide, GuidePage, MenuItem } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useEffect } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import paymentsImage from './images/payments.png';
+import whatsappImage from './images/whatsapp.png';
+
+function WhatsNewMenuItem() {
+	const { toggleWhatsNew } = useDispatch( 'whats-new' );
+	const isActive = useSelect( ( select ) => select( 'whats-new' ).isWhatsNewActive() );
+	const whatsNewPages = getWhatsNewPages();
+
+	return (
+		<>
+			<Fill name="ToolsMoreMenuGroup">
+				<MenuItem onClick={ () => toggleWhatsNew() }>
+					{ __( "What's New", 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'What's new' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/whats-new)